pulser-core/pulser/json/abstract_repr/serializer.py

```diff
@@ -51,25 +52,35 @@ def abstract_repr(name: str, *args: Any, **kwargs: Any) -> dict[str, Any]:
         signature = SIGNATURES[name]
     except KeyError:
         raise ValueError(f"No signature found for '{name}'.")
+    arg_as_kwarg: tuple[str, ...] = tuple()
     if len(args) < len(signature.pos):
-        raise ValueError(
-            f"Not enough positional arguments given for '{name}' (expected "
-            f"{len(signature.pos)}, got {len(args)})."
-        )
+        arg_as_kwarg = signature.pos[len(args) :]
```

Comment:
Took me sometime to understand what was happening here. Maybe we could add a comment explaining that all the positional arguments that were not passed as args should be looked for in the kwrags ?

Reply:
Sure, good idea!